cli/internal/server/turbo_grpc.pb.go

```diff
@@ -0,0 +1,215 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
I'm struggling to come up with a good opinion on this. There is definitely an argument for "don't check in generated code" on the one hand, but on the other hand, it doesn't seem great to have to run a generate step before vscode will be happy. `make turbo` is properly wired up so that it will work either way.

My inclination is to bias towards ease-of-contribution, since I don't think we're particularly susceptible to the problems with checked-in generated code (it's stale -- auto run by our build process, people modify it -- we can watch for it in reviews). But I don't feel super strongly about it.

Reply:
We could at least partially alleviate the issue using VS Code's `tasks.json` (which we should do a better job leveraging at some point anyway).

They have a [`"runOptions": { "runOn": "folderOpen" }"`](https://code.visualstudio.com/docs/editor/tasks#_run-behavior) that would make things happy. The other thing is I'm not sure if it will ever generate platform-specific code?